### DISCUSSION TITLE
Bin Size Stamp on Bottom

### DISCUSSION THREAD

**Discussion by walloutlet** [Le0Michine/FusionGridfinityGenerator#101] [Ideas]
I love this generator and use it all the time.  One of the things that I started doing because I have so many bins is started to stamp the bin size on the bottom so I could keep track of what is what.  Mainly so that I can get the right bins to fit into the rugged cases that are out there.

Could we put this as a check box inside the generator that would emboss the bin size on the bottom?  See picture for reference.

![24-05-12 14-21-30 7563](https://github.com/Le0Michine/FusionGridfinityGenerator/assets/117780452/ad20fc4c-80f1-417d-9be0-27b650e620b2)

**Comment by Jeppedy** [comment]
I can usually figure out bin Row x Col, but yeah, height would be nice.  You've got me thinking now...  I'll just write it on the bottom as needed until I find a better, easier solution!
